mergify_engine/tests/functional/actions/test_queue.py

```diff
@@ -2585,6 +2587,766 @@ async def test_queue_manual_merge(self):
 
         await self._assert_cars_contents(q, None, [])
 
+    async def test_create_queue_freeze(self):
```

Comment:
this test is very very very long
it'd be better to split in several `test_*` methods that test the different scenarios

Reply:
Might need some more indication on the best way to separate test scenarios, I tried to it but I'm sure it can be improve